src/types/index.ts

```diff
@@ -151,6 +151,7 @@ export interface IAccount {
   secretKey: Uint8Array
   showed: boolean
   type: string
+  protocol: string,
```

Comment:
The lines (properties) of the interface should be ended with semi (`;`).

Reply:
Fixed it.